src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -148,6 +148,16 @@
   private static final long ROTATE_INTERVAL_MS_DEFAULT = -1L;
   private static final String ROTATE_INTERVAL_MS_DISPLAY = "Rotate Interval (ms)";
 
+  public static final String ROTATE_SCHEDULE_INTERVAL_MS_CONFIG = "rotate.schedule.interval.ms";
+  private static final String ROTATE_SCHEDULE_INTERVAL_MS_DOC =
+          "The time interval in milliseconds to periodically invoke file commits. This configuration ensures that "
+          + "file commits are invoked every configured interval. Time of commit will be adjusted to 00:00 of selected timezone. "
```

Comment:
I mean that in case of interval that does not fit to a day (like 7 hours for example) it will be forced to fit into a day. For 7 hours interval it will trigger commit at 00:00, 07:00, 14:00, 21:00 and 00:00 of next day. Any ideas how to explain it in one sentence?